xml/System.Net.Http/HttpRequestException.xml

```diff
@@ -141,5 +141,80 @@
+      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception, statusCode As Nullable(Of System.Net.HttpStatusCode)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; HttpRequestException(System::String ^ message, Exception ^ inner, Nullable<System::Net::HttpStatusCode> statusCode);" />
+      <MemberSignature Language="F#" Value="new System.Net.Http.HttpRequestException : string * Exception * Nullable<System.Net.HttpStatusCode> -&gt; System.Net.Http.HttpRequestException" Usage="new System.Net.Http.HttpRequestException (message, inner, statusCode)" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Net.Http</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="message" Type="System.String" />
+        <Parameter Name="inner" Type="System.Exception" />
+      </Parameters>
+      <Docs>
+        <param name="message">A message that describes the current exception.</param>
+        <param name="inner">The inner exception.</param>
+        <param name="statusCode">The HTTP status code.</param>
+        <summary>Initializes a new instance of the <see cref="HttpRequestException" /> class with a specific message that describes the current exception, an inner exception, and an HTTP status code.</summary>
+      </Docs>
+    </Member>
+    <Member MemberName="StatusCode">
+      <MemberSignature Language="C#" Value="public System.Net.HttpStatusCode? StatusCode { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Nullable<valuetype System.Net.HttpStatusCode> StatusCode" />
+      <MemberSignature Language="DocId" Value="P:System.Nullable{System.Net.HttpStatusCode}" />
+      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property StatusCode As Nullable(Of System.Net.HttpStatusCode)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property Nullable<System::Net::HttpStatusCode> ^ StatusCode { Nullable<System::Net::HttpStatusCode> ^ get(); };" />
+      <MemberSignature Language="F#" Value="member this.StatusCode : Nullable<System.Net.HttpStatusCode>" Usage="System.Net.Http.HttpRequestException.StatusCode" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Net.Http</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Nullable{System.Net.HttpStatusCode}</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the HTTP status code to be returned with the exception.</summary>
+        <value>An HTTP status code if the exception represents a non-successful result, otherwise <c>null</c>.</value>
+      </Docs>
```

Comment:
You could do something like this, and it would display similar to [this property](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.webcontrols.boundfield.htmlencodeformatstring?view=netframework-4.8#System_Web_UI_WebControls_BoundField_HtmlEncodeFormatString).

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks 
  
> [!NOTE]
> This property is new in .NET 5.0.  
  
 ]]></format>
        </remarks>
      </Docs>
```

Reply:
@alnikola will you get a chance to continue working on this PR?